samples/packages/pulumi-typescript/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './package-config';
```

Comment:
See my comments below for possible other files to be included.

Reply:
ah, ok so that clarifies that comment.

Good catch. Sorted.

I'd assumed for some reason they get pulled through, which of source doesn't make sense.